ui/article-renderer/src/main/res/layout/list_item_article.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
`RelativeLayout` would give better performance for this layout since you aren't using any automatic size distribution that `LinearLayout` is optimized for.